docs/source/reference/samplers/index.rst

```diff
@@ -11,6 +11,60 @@ The :mod:`~optuna.samplers` module defines a base class for parameter sampling a
+| Multivariate optimization        |       ▲       |     ▲        |     ✅      |      ✅      |                   ▲                    |     ▲      |       ✅       |
++----------------------------------+---------------+--------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Conditional search space         |       ✅      |     ▲        |     ✅      |      ▲       |                   ▲                    |     ▲      |       ▲        |
++----------------------------------+---------------+--------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Multi-objective optimization     |       ✅      |     ▲        |     ✅      |      ❌      |       ✅(▲ for single-objective)       |     ▲      |       ✅       |
++----------------------------------+---------------+--------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Batch optimization               |       ✅      |     ✅       |     ✅      |      ✅      |                   ✅                   |     ✅     |       ▲        |
++----------------------------------+---------------+--------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Distributed optimization         |       ✅      |     ✅       |     ✅      |      ✅      |                   ✅                   |     ✅     |       ▲        |
++----------------------------------+---------------+--------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Constrained optimization         |       ❌      |     ❌       |     ✅      |      ❌      |                   ✅                   |     ❌     |       ✅       |
++----------------------------------+---------------+--------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Time complecity (per trial) (**) |      O(1)     |    O(1)      |    O(n)     |    O(d^3)    |                 O(mp^2)                |    O(1)    |     O(n^3)     |
++----------------------------------+---------------+--------------+-------------+--------------+----------------------------------------+------------+----------------+
+| Recommended budgets (#trials)    | as many as    | < number of  | 100 ~ 1000  | 1000 ~ 10000 |                100 ~ 10000             | as many as |    10 ~ 100    |
+|                                  | one likes     | possible     |             |              |                                        | one likes  |                |
+|                                  |               | combinations |             |              |                                        |            |                |
```

Comment:
Thanks for the update! Not sure but how about updating recommended budgets like the following?

* GridSampler: `number of combinations`, not `< number of ...`
* TPESampler: `10~100 × #params`
* CmaEsSampler: `100~1000 x #params`
* NSGAIISampler: `10~1000 × #params`
* BoTorchSampler: `10~50 × #params`

Reply:
(please ignore this comment as it would be too complicated to add but `#objectives` is an important factor to decide the number of trials in multi-objective optimization)

Reply:
There is no doubt that the number of parameters and the number of objectives affect the number of trials that can be run, but I recognize that a quantitative discussion of that effect is an open question. I would prefer not to express such information in this table as the opinion of the Optuna developers. As an alternative, how about a note that "budget depends on the number of parameters and the number of objectives."?

Reply:
Thanks for the consideration. It was not a strong opinion since there is no clear answers here. Please ignore my suggestions except for GridSampler.

As for GridSampler, does my suggestion sound reasonable?

Reply:
> As for GridSampler, does my suggestion sound reasonable?

Yes, I agree with you. Fixed!